File: project_examples/go/project_main.go
package main

import "fmt"

func main() {
    items := []int{12, 18, 21, 9, 31}
    total := 0
    for _, value := range items { total += value }
    fmt.Println("Total:", total)
    fmt.Printf("Average: %.2f\n", float64(total)/float64(len(items)))
}

File: project_examples/csharp/project_main.cs
using System;
class Program {
    static void Main() {
        int[] items = { 12, 18, 21, 9, 31 };
        int total = 0;
        foreach (int value in items) total += value;
        Console.WriteLine($"Total: {total}");
        Console.WriteLine($"Average: {total / (double)items.Length:F2}");
    }
}
